runtime/jcl/common/mgmtosext.c

```diff
@@ -98,10 +101,14 @@ Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getSyste
 			portLibraryStatus = -3;
 			break;
 		case OMRPORT_ERROR_INSUFFICIENT_DATA:
-			portLibraryStatus =
-				J9_ARE_ALL_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_CPU_LOAD_COMPATIBILITY)
-				? 0
-				: -1;
+			if (J9_ARE_ALL_BITS_SET(vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_CPU_LOAD_COMPATIBILITY)
+				&& !retried
+			) {
```

Comment:
can we simplify this by just calling `omrsysinfo_get_CPU_load` a second time if `J9_EXTENDED_RUNTIME2_CPU_LOAD_COMPATIBILITY` in the `OMRPORT_ERROR_INSUFFICIENT_DATA` path?